src/main/java/core/basesyntax/CalculationFruits.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import core.basesyntax.service.CalculationFile;
+import core.basesyntax.service.FruitsDto;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CalculationFruits implements CalculationFile {
+    Map<String, Integer> calculation = new HashMap<>();
+
+    @Override
+    public Map<String, Integer> getCalculation(List<FruitsDto> data) {
+        for (FruitsDto value : data) {
+            if (calculation.containsKey(value.getFruit())) {
+                if (value.getType().contains("b")) {
```

Comment:
expecting to see it in the SOLID style. Will review your code after SOLID will be implemented